pkgs/development/libraries/unicorn-emu/default.nix

```diff
@@ -1,18 +1,25 @@
-{ stdenv, fetchurl, pkgconfig, python }:
+{ stdenv, fetchurl, pkgconfig, python, cmocka, hexdump, setuptools }:
 
 stdenv.mkDerivation rec {
   pname = "unicorn-emulator";
-  version = "1.0.1";
+  version = "1.0.2-rc4";
```

Comment:
We typically allow only final releases.

Reply:
That's ok with me, but if so then we need to downgrade pwntools instead. 

Their last final release was more than 3 years ago, and it feels like their release candidates are more like point releases for other software: https://github.com/unicorn-engine/unicorn/releases